nautobot/extras/api/serializers.py

```diff
@@ -107,7 +110,29 @@
 #
 
 
-class NautobotModelSerializer(RelationshipModelSerializerMixin, CustomFieldModelSerializer, ValidatedModelSerializer):
+class NotesSerializerMixin(BaseModelSerializer):
+    """Extend Serializer with a `notes` field."""
+
+    notes = serializers.SerializerMethodField()
```

Comment:
Should this be `notes_url`?

Reply:
There was a reason I used `notes` instead of `notes_url`. I was getting an error about `notes_url` not being a valid field, but I will test again.

Reply:
Not getting any obvious failures now, I might have been trying to use `notes_url` before I switched to the SerializerMethodField.